R/tune_sim_anneal.R

```diff
@@ -294,6 +306,7 @@ tune_sim_anneal_workflow <-
 
     metrics <- tune::check_metrics(metrics, object)
     metrics_name <- names(attr(metrics, "metrics"))[1]
+    metrics_time <- eval_time[1]
```

Comment:
So if `eval_time` is a vector, we only ever use the first element? I'd say we should document that.

Reply:
No, that object is used to guide the optimization or racing. We compute all of them but use one for analysis.